HelpSource/editor.js

```diff
@@ -38,7 +38,10 @@ const init = () => {
             lineWrapping: true,
             viewportMargin: Infinity,
             extraKeys: {
-                'Shift-Enter': evalLine
+                // noop: prevent both codemirror and the browser to handle Shift-Enter
+                'Shift-Enter': ()=>false 
```

Comment:
missing a comma here

Reply:
yesss, thanks for catching it! btw I changed to ()=>{} which looks like a more proper noop